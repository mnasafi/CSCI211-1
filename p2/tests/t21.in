rating
title 00
url 00
comment 00
20.37
3
title 01
url 01
comment 01
5.94
5
title 02
url 02
comment 02
91.5
5
title 03
url 03
comment 03
38.54
3
title 04
url 04
comment 04
40.84
2
title 05
url 05
comment 05
53.82
2
title 06
url 06
comment 06
71.48
1
title 07
url 07
comment 07
65.78
5
title 08
url 08
comment 08
77.26
1
title 09
url 09
comment 09
96.85
1
title 10
url 10
comment 10
33.80
2
title 11
url 11
comment 11
27.59
1
title 12
url 12
comment 12
32.13
1
title 13
url 13
comment 13
38.27
3
title 14
url 14
comment 14
75.80
5
title 15
url 15
comment 15
9.98
2
title 16
url 16
comment 16
9.6
4
title 17
url 17
comment 17
87.51
4
title 18
url 18
comment 18
14.22
5
title 19
url 19
comment 19
99.83
5
title 20
url 20
comment 20
31.75
4
title 21
url 21
comment 21
90.86
5
title 22
url 22
comment 22
55.1
3
title 23
url 23
comment 23
82.6
2
title 24
url 24
comment 24
14.1
1
title 25
url 25
comment 25
12.35
1
title 26
url 26
comment 26
18.21
4
title 27
url 27
comment 27
70.30
3
title 28
url 28
comment 28
92.3
1
title 29
url 29
comment 29
27.4
1
title 30
url 30
comment 30
2.48
3
title 31
url 31
comment 31
88.76
4
title 32
url 32
comment 32
41.82
3
title 33
url 33
comment 33
48.71
4
title 34
url 34
comment 34
1.97
3
title 35
url 35
comment 35
36.50
4
title 36
url 36
comment 36
57.10
4
title 37
url 37
comment 37
39.31
3
title 38
url 38
comment 38
94.38
1
title 39
url 39
comment 39
98.58
5
title 40
url 40
comment 40
46.80
1
title 41
url 41
comment 41
74.84
1
title 42
url 42
comment 42
56.77
1
title 43
url 43
comment 43
27.24
5
title 44
url 44
comment 44
77.4
5
title 45
url 45
comment 45
27.46
5
title 46
url 46
comment 46
37.52
4
title 47
url 47
comment 47
68.75
2
title 48
url 48
comment 48
6.19
1
title 49
url 49
comment 49
16.6
3
title 50
url 50
comment 50
48.20
1
title 51
url 51
comment 51
32.88
4
title 52
url 52
comment 52
61.43
2
title 53
url 53
comment 53
85.38
5
title 54
url 54
comment 54
90.33
1
title 55
url 55
comment 55
88.79
4
title 56
url 56
comment 56
92.9
1
title 57
url 57
comment 57
19.99
5
title 58
url 58
comment 58
90.75
3
title 59
url 59
comment 59
49.97
5
title 60
url 60
comment 60
69.17
5
title 61
url 61
comment 61
57.82
3
title 62
url 62
comment 62
52.72
4
title 63
url 63
comment 63
43.70
5
title 64
url 64
comment 64
28.95
4
title 65
url 65
comment 65
7.47
4
title 66
url 66
comment 66
68.26
2
title 67
url 67
comment 67
67.93
3
title 68
url 68
comment 68
43.79
2
title 69
url 69
comment 69
92.71
5
title 70
url 70
comment 70
9.83
1
title 71
url 71
comment 71
91.42
4
title 72
url 72
comment 72
16.36
1
title 73
url 73
comment 73
86.39
3
title 74
url 74
comment 74
33.79
5
title 75
url 75
comment 75
33.59
3
title 76
url 76
comment 76
59.79
1
title 77
url 77
comment 77
4.91
2
title 78
url 78
comment 78
83.73
4
title 79
url 79
comment 79
7.96
3
title 80
url 80
comment 80
42.16
1
title 81
url 81
comment 81
36.56
4
title 82
url 82
comment 82
72.56
4
title 83
url 83
comment 83
12.56
1
title 84
url 84
comment 84
43.50
1
title 85
url 85
comment 85
2.19
3
title 86
url 86
comment 86
82.47
3
title 87
url 87
comment 87
25.94
1
title 88
url 88
comment 88
50.55
5
title 89
url 89
comment 89
46.11
3
title 90
url 90
comment 90
14.58
4
title 91
url 91
comment 91
22.13
4
title 92
url 92
comment 92
79.63
3
title 93
url 93
comment 93
35.76
1
title 94
url 94
comment 94
38.79
3
title 95
url 95
comment 95
9.14
4
title 96
url 96
comment 96
8.81
2
title 97
url 97
comment 97
2.58
3
title 98
url 98
comment 98
58.90
2
title 99
url 99
comment 99
69.4
5
