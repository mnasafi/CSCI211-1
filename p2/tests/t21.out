title 01, url 01, comment 01, 5.94, *****
title 02, url 02, comment 02, 91.5, *****
title 07, url 07, comment 07, 65.78, *****
title 14, url 14, comment 14, 75.8, *****
title 18, url 18, comment 18, 14.22, *****
title 19, url 19, comment 19, 99.83, *****
title 21, url 21, comment 21, 90.86, *****
title 39, url 39, comment 39, 98.58, *****
title 43, url 43, comment 43, 27.24, *****
title 44, url 44, comment 44, 77.4, *****
title 45, url 45, comment 45, 27.46, *****
title 53, url 53, comment 53, 85.38, *****
title 57, url 57, comment 57, 19.99, *****
title 59, url 59, comment 59, 49.97, *****
title 60, url 60, comment 60, 69.17, *****
title 63, url 63, comment 63, 43.7, *****
title 69, url 69, comment 69, 92.71, *****
title 74, url 74, comment 74, 33.79, *****
title 88, url 88, comment 88, 50.55, *****
title 99, url 99, comment 99, 69.4, *****
title 16, url 16, comment 16, 9.6, ****
title 17, url 17, comment 17, 87.51, ****
title 20, url 20, comment 20, 31.75, ****
title 26, url 26, comment 26, 18.21, ****
title 31, url 31, comment 31, 88.76, ****
title 33, url 33, comment 33, 48.71, ****
title 35, url 35, comment 35, 36.5, ****
title 36, url 36, comment 36, 57.1, ****
title 46, url 46, comment 46, 37.52, ****
title 51, url 51, comment 51, 32.88, ****
title 55, url 55, comment 55, 88.79, ****
title 62, url 62, comment 62, 52.72, ****
title 64, url 64, comment 64, 28.95, ****
title 65, url 65, comment 65, 7.47, ****
title 71, url 71, comment 71, 91.42, ****
title 78, url 78, comment 78, 83.73, ****
title 81, url 81, comment 81, 36.56, ****
title 82, url 82, comment 82, 72.56, ****
title 90, url 90, comment 90, 14.58, ****
title 91, url 91, comment 91, 22.13, ****
title 95, url 95, comment 95, 9.14, ****
title 00, url 00, comment 00, 20.37, ***
title 03, url 03, comment 03, 38.54, ***
title 13, url 13, comment 13, 38.27, ***
title 22, url 22, comment 22, 55.1, ***
title 27, url 27, comment 27, 70.3, ***
title 30, url 30, comment 30, 2.48, ***
title 32, url 32, comment 32, 41.82, ***
title 34, url 34, comment 34, 1.97, ***
title 37, url 37, comment 37, 39.31, ***
title 49, url 49, comment 49, 16.6, ***
title 58, url 58, comment 58, 90.75, ***
title 61, url 61, comment 61, 57.82, ***
title 67, url 67, comment 67, 67.93, ***
title 73, url 73, comment 73, 86.39, ***
title 75, url 75, comment 75, 33.59, ***
title 79, url 79, comment 79, 7.96, ***
title 85, url 85, comment 85, 2.19, ***
title 86, url 86, comment 86, 82.47, ***
title 89, url 89, comment 89, 46.11, ***
title 92, url 92, comment 92, 79.63, ***
title 94, url 94, comment 94, 38.79, ***
title 97, url 97, comment 97, 2.58, ***
title 04, url 04, comment 04, 40.84, **
title 05, url 05, comment 05, 53.82, **
title 10, url 10, comment 10, 33.8, **
title 15, url 15, comment 15, 9.98, **
title 23, url 23, comment 23, 82.6, **
title 47, url 47, comment 47, 68.75, **
title 52, url 52, comment 52, 61.43, **
title 66, url 66, comment 66, 68.26, **
title 68, url 68, comment 68, 43.79, **
title 77, url 77, comment 77, 4.91, **
title 96, url 96, comment 96, 8.81, **
title 98, url 98, comment 98, 58.9, **
title 06, url 06, comment 06, 71.48, *
title 08, url 08, comment 08, 77.26, *
title 09, url 09, comment 09, 96.85, *
title 11, url 11, comment 11, 27.59, *
title 12, url 12, comment 12, 32.13, *
title 24, url 24, comment 24, 14.1, *
title 25, url 25, comment 25, 12.35, *
title 28, url 28, comment 28, 92.3, *
title 29, url 29, comment 29, 27.4, *
title 38, url 38, comment 38, 94.38, *
title 40, url 40, comment 40, 46.8, *
title 41, url 41, comment 41, 74.84, *
title 42, url 42, comment 42, 56.77, *
title 48, url 48, comment 48, 6.19, *
title 50, url 50, comment 50, 48.2, *
title 54, url 54, comment 54, 90.33, *
title 56, url 56, comment 56, 92.9, *
title 70, url 70, comment 70, 9.83, *
title 72, url 72, comment 72, 16.36, *
title 76, url 76, comment 76, 59.79, *
title 80, url 80, comment 80, 42.16, *
title 83, url 83, comment 83, 12.56, *
title 84, url 84, comment 84, 43.5, *
title 87, url 87, comment 87, 25.94, *
title 93, url 93, comment 93, 35.76, *
